### ISSUE TITLE
Suggestion implementation of procedure to pause/wait random time intervals

### ISSUE THREAD

**Issue by neverdie11** [fkapitalism/HExBot#34]
worked on the webcrawler and implemented a procedure to wait random times before switching to certain pages during crawling, to make crawler behaviour look more human.

the pauses can be called in the sequence. i think this is better than to use {wait:x}.

this can easily be used in the other modules, e.g. puzzle solver and mission solver.

**Comment by neverdie11** [comment]
[update_01.zip](https://github.com/fkapitalism/HExBot/files/1070968/update_01.zip)

**Comment by MentalMushrooms** [comment]
@fkapitalism Please read this post, may be useful.

This could help with the bot, great idea.

Maybe this can be toggled as an option for webcrawling.

Such as...

Webcrawler : Mode : 

                                Human Mode ( slower but more "stable" )
             
                                Bot Mode ( Faster but more "noticeable" )

**Comment by neverdie11** [comment]
good idea, might work on a (Checkbox Option) if i manage to wrap my head artound it
